route/build/cpl/RtmFileUtils.F90

```diff
@@ -149,7 +149,7 @@ subroutine opnfil (locfn, iun, form)
      ! !ARGUMENTS:
      implicit none
      character(len=*), intent(in):: locfn  !file name
-     integer, intent(in):: iun             !fortran unit number
+     integer            :: iun             !fortran unit number
```

Comment:
is this supposed to be input intent??  or inout??

Reply:
It seems to need to be inout, so I removed the intent. But, I could be explicit to say

` integer, intent(inout):: iun`

if you prefer. That might be better because it means the intent was thought about. 

And actually, maybe what it should be is: intent(out), since that's the way it's really working. So I'll try that...